src/routes/blog/post/announcing-messaging-set-up-a-messaging-service

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Announcing messaging: set up a full-functioning messaging service"
+description: 
+date: 2024-02-26
+cover: /images/blog/messaging-announcement.png
+timeToRead: 4
+author: eldad-fux
+category: engineering
```

Comment:
It should be `product` or something. Engineering needs to be much more technical stuff about our internal processes. We need to have a clear list of categories for the future.